### ISSUE TITLE
Change default number of example languages

### ISSUE THREAD

**Issue by nsundbaum** [Mermade/widdershins#6]
Hi,

First of all, thanks for this very nice framework! It is a perfect fit in our API doc pipeline.

I've been playing around with limiting the number of languages show in the languages tabs, by using configuration similar to the following:

```
options.language_tabs = [{'shell': 'Shell'},{'http': 'HTTP'}];
```

This hides other languages in the top tab menu, but the code samples for those other languages are still showing in the right column, which makes that entire column look very strange. Am I doing something wrong or is this an issue with the framework? If needed, I'd be happy to provide a pull request.

Also, would be interested to hear about any plans for future features/improvements that you might have for this framework.

**Comment by MikeRalphson** [comment]
Hi @nsundbaum - thank you for the kind comments.

The issue was a mistake on my part, I didn't foresee limiting the built-in language defaults in this way, only as an all-or-nothing setting...

I've pushed a fix which should allow for your use-case. Please let me know if you are working from the npm package and I can publish a new version.

As to future plans, I am hoping to add replaceable templates for at least some of the sections of markdown generated, to allow for customisation, translation and replacing the generic code samples. Other ideas gratefully received!

**Comment by nsundbaum** [comment]
Sounds cool, thanks for the swift reply and fix!

**Comment by MikeRalphson** [comment]
Code samples are now templated with doT.js. `header` and `authentication_none` tempates also added, more to come, with the aim of allowing all English text to be replaced.

**Comment by nsundbaum** [comment]
That's really good stuff! 

What would be the recommended approach if I want to override any of them (I work from the npm package)?

**Comment by MikeRalphson** [comment]
I think the best way might be to add an option  specifying a user_templates directory. Any templates present there would then override the defaults. 

I'm open to other approaches though. I'll let you know when the npm package is updated.

**Comment by MikeRalphson** [comment]
@nsundbaum I've published v1.0.6 with the templating functionality. Please let me know if there are any problems.
